mw-webapp/src/component/table/renderCellValue/handlers/handleDoubleClick.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Handle actions on double click
+ * @param {boolean} callback takes as argument boolean value
+ */
+export const handleDoubleClick = (callback: (arg: boolean) => void) => {
```

Comment:
It looks like a function which could call everything with true arg. Call it trueCaller instead)

(Joke)

Reply:
So should I change it or not?

Reply:
I think so

Reply:
Renamed to setTrueValue